.env.docker-override

```diff
@@ -1,9 +1,6 @@
 # Overrides any values in .env that are being used for local development
 # Values here are used inside the contianers using this file
 DATA_DIR=/stored_data
-DATA_DIR_SLR=/stored_data/slr_data
-DATA_DIR_REC=/stored_data/rec_data
-DATA_DIR_MODEL_OUTPUT=/stored_data/model_output
```

Comment:
Associated with these changes, could you also ensure they're removed from `.env.template`, `docker-compose.yml,` and `docker-compose-dev.yml`.
Make sure you're up to date with the base branch when you do this.

Reply:
Done. Can you please review the changes? @LukeParky

Reply:
Looks like you aren't up to date with `otakaro_digital_twin` branch so I can't confirm it but it looks right to me. Once you merge in `otakaro_digital_twin` I'll do a full review